docs/syntax/types.md

```diff
@@ -299,7 +299,7 @@ below.
 
 - **Autoscoped Constructors:** Referencing constructors via their type name may
   lead to long and boilerplate code. To simplify referencing constructors when
-  the _context is known_ a special `~` syntax is supported. Should there be a
+  the _context is known_ a special `..` syntax is supported. Should there be a
```

Comment:
The example just below is still using the old syntax. Please update the example to be consistent.

Reply:
```ruby
describe (~Just 5)
```

Reply:
Thank you for catching my omission. Fixed in 1057376